src/game/client/components/netgui.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015 Henritees
+
+#ifndef GAME_CLIENT_COMPONENTS_NETGUI_H
+#define GAME_CLIENT_COMPONENTS_NETGUI_H
+
+#include <base/tl/array.h>
+#include <game/client/component.h>
+
+class CModAPIGui : public CComponent
```

Comment:
Conflicts are impossible since the client will _never_ include *anything* from the server,but you are right, it's better to rename it. Users and IDEs might get confused.